### PR TITLE
added logo to README file title

### PR DESCRIPTION
**Pull request by Jahkamso**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution. 

Please update with

1. format the file correctly, the logo code should be on it's own line not part of the heading
2. the logo should be small, not the the whole page (see screenshot)

Screenshot 2024-06-17 at 08 53 07

**Comment by rajneesh-18** [comment]
> Thank you for the contribution.
> 
> Please update with
> 
> 1. format the file correctly, the logo code should be on it's own line not part of the heading
> 2. the logo should be small, not the the whole page (see screenshot)
> 
> Screenshot 2024-06-17 at 08 53 07

Yeah, it is taking the whole space. Make it small.

**Comment by Jahkamso** [comment]
Working on it now

**Comment by Jahkamso** [comment]
I've resized the image

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Creators Registry
+# Creators Registry <img src="./public/logo.svg" width=35>
```
Comment (eddiejaoude) [inline README.md:1]: As mentioned in my review in point 1, it should not be part of the heading

```suggestion
# Creators Registry

<img src="./public/logo.svg" width=35>
```

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Review by rajneesh-18 — COMMENTED** [review]
That's great to see.

**Pull request merged**